imports.go

```diff
@@ -208,11 +208,20 @@ func listPackages() ([]string, error) {
 }
 
 func isOwnPackage(path string) bool {
+	// workdir is without trailing slash, to make sure we don't exclude
+	// packages that have the same folder prefix, suffix with dir separator
+	wdir := workdir + string(os.PathSeparator)
+
 	for _, gopath := range filepath.SplitList(os.Getenv("GOPATH")) {
-		if strings.HasPrefix(filepath.Join(gopath, "src", path), workdir) {
+		p := filepath.Join(gopath, "src", path)
```

Comment:
let's use more obvious name for this variable, for example, projectPath or projectDir or fullPath, but not just `p`. Explicit is better than implicit.